repo/packages/R/redis/0/package.json

```diff
@@ -0,0 +1,19 @@
+{
+ "packagingVersion": "3.0",
+ "name": "redis",
+ "version": "0.0.1",
+ "maintainer": "joerg@mesosphere.io",
+ "description": "Single redis container.",
+ "framework": false,
+ "tags": ["mesosphere", "service", "redis"],
+ "licenses": [
+   {
+     "name": "BSD",
+     "url": "http://redis.io/topics/license"
+   }
+ ],
+ "website": "http://redis.io",
+ "postInstallNotes": "Redis installed!",
+ "preInstallNotes": "Redis requires a single node with 300MB of RAM and 0.1 CPUs. ",
```

Comment:
Add this line to the beginning of `preInstallNotes`: `This DC/OS Service is currently EXPERIMENTAL. There may be bugs, incomplete features, incorrect documentation, or other discrepancies.`